apps/api/lib/api/web/controllers/game_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule API.Web.GameController do
+  use API.Web, :controller
+  plug Guardian.Plug.EnsureAuthenticated, handler: __MODULE__
```

Comment:
should we move this into a different router pipeline and use it for all the routes that require authentication?